crates/runix/src/installable.rs

```diff
@@ -22,9 +23,15 @@ use crate::flake_ref::{FlakeRef, ParseFlakeRefError};
 static VALID_ATTRIBUTE: Lazy<Regex> =
     Lazy::new(|| Regex::new("^([a-zA-Z0-9-._~!$&'()*+,;=:%@?/ ]*)$").unwrap());
 
-/// A simplified installable representation
+#[derive(Clone, Debug, Display, Eq, PartialEq)]
```

Comment:
We should derive `From` here as well, which makes it easier to fix things downstream

Reply:
`FromStr` is a bit awkward to implement for these, since file and expressions (to avoid ambiguity) already require explicit flags.
However, we might want to at least implement `FromStr` to parse FlakeAttributes and StorePaths